official-allow_functionality.json

```diff
@@ -229,8 +229,43 @@
 		{"o":{"h":"*.codepen.io"},"d":{"h":"*.s3-us-west-2.amazonaws.com"}},
 		{"o":{"h":"*.codepen.io"},"d":{"h":"*.cloudflare.com"}},
 		{"o":{"h":"*.hansje.net"},"d":{"h":"*.cargocollective.com"}},
-		{"o":{"h":"*.indiegogo.com"},"d":{"h":"*.iggcdn.com"}}
+		{"o":{"h":"*.indiegogo.com"},"d":{"h":"*.iggcdn.com"}},
+		{"o":{"h":"*.eurobuch.com"},"d":{"h":"*.lehmanns.de"}},
+		{"o":{"h":"*.emp.de"},"d":{"h":"*.acfrg.com"}},
+		{"o":{"h":"*.microsoft.com"},"d":{"h":"*.s-msft.com"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.trackersimulator.org"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.eviltracker.net"}},
```

Comment:
Yes. They run special tests at `panopticlick.eff.org`, therefore `eviltracker.net` and `trackersimulator.org` are needed. If one don't want to run those tests or are not reliant on true(?) response, we could just delete these rules (or even blacklist them).